mw-webapp/src/styles/_variables.scss

```diff
@@ -0,0 +1,4 @@
+$color-btn: rgb(177, 162, 149);
+$color-main: rgb(246, 246, 150);
+$color-shadow: rgb(77, 77, 77);
+$color-active: rgb(255, 149, 0);
```

Comment:
1. Why consts are not in index? Do we need separate folder just for one file with consts? What do you think?
2. Create structure for consts

Reply:
Done.